### PR TITLE
Add step to setting up environment

### PR DESCRIPTION
**Pull request by mr-bjerre**
The test files are downloaded using git submodules. This is not explained in `README.md`. This PR adds that step.

### CODE REVIEW

**Comment by mr-bjerre** [comment]
We could also add devcontainer.json and Dockerfile so you can easily start a development setup inside a docker container using VSCodes Remote Container. 

Let me know what you think - I can quickly set that up

File: README.md [hunk 1]
```diff
@@ -35,7 +35,8 @@ More advanced examples in [the "examples" folder](./examples)
 2. Run `source .venv/bin/activate` to go into that virtual environment
 3. Run `pip install -r requirements_dev.txt` to install all of the dependencies in `requirements.txt` (which includes `pytest`)
 4. Run `npm i @scion-scxml/test-framework` to download scxml files
```
Comment (davidkpiano) [inline README.md:37]:
```suggestion
```

We can delete this

File: README.md [hunk 2]
```diff
@@ -35,7 +35,8 @@ More advanced examples in [the "examples" folder](./examples)
...
-5. Run `pytest` to run the tests! 👩‍🔬
+5. Make sure test files are present and up to date, run `git submodule init` (first time) and `git submodule update`
+6. Run `pytest` to run the tests! 👩‍🔬
```
Comment (davidkpiano) [inline README.md:39]:
```suggestion
1. Make sure test files are present and up to date by running `git submodule init` (first time) and `git submodule update`
1. Run `pytest` to run the tests! 👩‍🔬
```

**Comment by mr-bjerre** [comment]
Ready to squash merge - let me know if you'd like a PR for VSCode Remote Container as well (as an option for developers to quickly get set up)

**Comment by davidkpiano** [comment]
> Ready to squash merge - let me know if you'd like a PR for VSCode Remote Container as well (as an option for developers to quickly get set up)

I think that's a great idea 👍

**Pull request merged**